### PR TITLE
fix organization tooltip in execution environment edit form

### PR DESCRIPTION
**Pull request by daphnemaeve**
Jira issue: https://issues.redhat.com/browse/AAP-24951

Before:
Screenshot 2024-06-06 at 10 23 00 PM

After:
Screenshot 2024-06-06 at 10 22 42 PM

### CODE REVIEW

**Comment by jerabekjiri** [comment]
Edit EE screen shows this message for empty organization in the old UI, should we add it?
Screenshot from 2024-06-07 13-32-30

File: frontend/awx/administration/execution-environments/ExecutionEnvironmentForm.tsx [hunk 1]
```diff
@@ -217,11 +217,19 @@ function ExecutionEnvironmentInputs(props: {
         label={t('Description')}
         placeholder={t('Enter a description')}
         isDisabled={props?.executionEnv?.managed || false}
+        helperText={t(
+          'Leave this field blank to make the execution environment globally available.'
```
Comment (jerabekjiri) [inline frontend/awx/administration/execution-environments/ExecutionEnvironmentForm.tsx:221]: Maybe a mistake? Description shouldn't have this

Comment (daphnemaeve) [reply frontend/awx/administration/execution-environments/ExecutionEnvironmentForm.tsx]: agreed, thanks for catching that. it should be fixed now!

**Review by skochay330 — APPROVED** [review]
LGTM!

**Comment by nixocio** [comment]
As a team we decided to merge this one, since the test failures are not related.

**Pull request merged**